### PR TITLE
fix-popover-reference-display

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/js/components/PopOver/PopOver.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
-	<div>
-		<div v-if="triggerAction === POP_OVER_TRIGGER_ACTIONS.NONE">
+	<span>
+		<span v-if="triggerAction === POP_OVER_TRIGGER_ACTIONS.NONE">
```
Comment (kvas-damian) [inline lib/js/components/PopOver/PopOver.vue:3]: can we use `<template v-if>` to avoid unnecessary DOM nodes which can break styles?

Comment (rogatty) [reply lib/js/components/PopOver/PopOver.vue:3]: I think we can even use `<slot v-if`

**Review by kvas-damian — APPROVED** [review]

File: lib/js/components/PopOver/PopOver.vue [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <template>
-	<div>
-		<div v-if="triggerAction === POP_OVER_TRIGGER_ACTIONS.NONE">
+	<span>
```
[the remark this answers is not in the document]
Comment (pioziol123) [reply lib/js/components/PopOver/PopOver.vue:2]: linter requires single root element
Zrzut ekranu 2024-05-27 o 08 45 55

**Review by kvas-damian — APPROVED** [review]

File: lib/js/components/PopOver/PopOver.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (rogatty) [reply lib/js/components/PopOver/PopOver.vue:2]: Oh, I think we have eslint misconfiguration in design-system, `slot` is allowed as a root element in the wnl-platform. I'll take a look into that, it's not a blocker.

**Pull request merged**